PEPit/examples/monotone_inclusions_variational_inequalities/frugal_resolvent_splitting.py

```diff
@@ -0,0 +1,277 @@
+from PEPit import PEP, null_point, Constraint
+from PEPit.primitive_steps import proximal_step
+from PEPit.functions import SmoothStronglyConvexFunction
+from numpy import array
+
+def wc_frugal_resolvent_splitting(L, W, lipschitz_values, mu_values, alpha=1, gamma=0.5, wrapper="cvxpy", solver=None, verbose=1):
+    """
+    Consider the the monotone inclusion problem
+
+    .. math:: \\mathrm{Find}\\, x:\\, 0 \\in \\sum_{i=1}^{n} A_i(x),
+
+    where :math:`A_i` is the subdifferential of an :math:`l_i`-Lipschitz smooth and :math:`\\mu_i`-strongly convex function for all :math:`i \\leq n`. 
```

Comment:
Thank you for the review and the inputs! I've incorporated the suggested changes into the latest push. One note here - while I chose to use the SmoothStronglyConvexFunction as the example here, the convergence of the algorithm holds in general for the monotone inclusion over any summation of maximal monotone operators. We can see nice linear rates in the reduced FRS for the entire LipschitzStronglyMonotoneOperator class as well, in which case the \in inclusion would apply, but I recognize that it makes sense to keep the text consistent with the chosen class in the example.

Reply:
Thank you for the comment.
- "for the entire LipschitzStronglyMonotoneOperator class as well, in which case the \in inclusion would apply": I do not understand this. If the operators are Lipschitz continuous, then they are single-valued, aren't they? If so, equality applies.
- For general maximally monotone operators, I agree with you: inclusion applies.
- Consistency is important, but there are still several choices here: monotone operators, Lipschitz strongly monotone operators, or gradient of smooth strongly convex functions. No option is better than the others, you decide what you prefer. Do you keep it as is?
- In case you consider the operators as gradients of strongly convex functions, why do you write  0 = sum of gradients in x, instead of writing that x minimizes the sum of functions ? I understand the operator's point of view, but it looks weird to me to write that you look for a stationary point of a strongly convex function, instead of directly talking of a minimizer.

- Quick question about the linear rate you mentioned: do you expect/observe the same rate on LipschitzStronglyMonotoneOperators as on gradients of SmoothStronglyConvexFunctions?

Reply:
You're absolutely right - if they're Lipschitz continuous, they are single valued. I should have said MonotoneOperator or StronglyMonotoneOperator. I modified the examples to take a PEP problem as the input so that the set of operators can be user defined, and now have the first example in the docstring using a MonotoneOperator and a SmoothStronglyConvexFunction. I kept the inclusion in the docstring and made the problem description more general. The user-defined PEP input also allowed me to set up the Douglas-Rachford tests. I used the same parameters that were provided in the existing Douglas-Rachford example. I put the test code in the `additional_complixified_examples_tests` folder, since it needed some additional imports and a wrapper around the example, and it looked like that was the design pattern in the tests folder for that type of test. 

For the question on the rates for LipschitzStronglyMonotoneOperator and SmoothStronglyConvexFunction --- I do not  expect or observe the same rates for the two classes. The contraction factor is not as strong (it's close to 1) for the LipschitzStronglyMonotoneOperator, which is what I would expect since it's a relaxation of the SmoothStronglyConvexFunction case.